CONTRIBUTING.md

```diff
@@ -33,6 +33,7 @@ To ensure a smooth code review and merging process please check if the Pull Requ
 - The GitHub Action pipeline concludes successfully:
   - The code follows the style guide used within this project. This can be tested locally by running: `npm run test:lint`.
   - The code passes all unit tests. This can be tested locally by running: `npm run test:unit`.
-- The change does not break the example e2e tests. This can be tested locally by running `npm run test:e2e:allure`.
+- The change does not break the example e2e tests. This can be tested locally by running `npm run test:e2e`. After running e2e tests, `allure` directory will have `allure-restults` that can be used to see example allure report. Please run `npm run allure` and check how final report will look like
```

Comment:
Spotted a tiny typo `allure-restults`. Can you edit this?

Reply:
Thank you very much for spotting it. Fixed!